layouts/page/event-one-pager.html

```diff
@@ -0,0 +1,30 @@
+{{ define "hero" }}
+    <header class="header-hero header-hero-glow">
+        <div class="header-hero-items container mx-auto w-full md:w-7/12">
+            <div class="header-hero-item">
+                <h1>{{ .Params.hero.title }}</h1>
+                <div class="text-base md:my-5">{{ .Params.hero.body | markdownify }}</div>
+                <div class="header-hero-actions w-full md:w-1/3">
+                    <a class="btn btn-lg" href="{{ .Params.hero.cta.url }}">
+                        {{ .Params.hero.cta.label }}
+                    </a>
+                </div>
+            </div>
+            <!-- <div class="hidden md:inline-block header-hero-item">
```

Comment:
Ah, I see. We should probably just remove this if we aren't going to use it.

Reply:
Yup good catch.